File: backend/src/main/java/Manager/Restaurant/mai/dto/OrderResponseDTO.java
package Manager.Restaurant.mai.dto;

import lombok.Builder;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
public class OrderResponseDTO {
    private Long orderId;
    private String orderStatus;
    private LocalDateTime orderDate;
    private LocalDateTime updatedAt;
    private BigDecimal totalAmount;
    private BigDecimal deliveryFee;
    private String restaurantAddress; // Địa chỉ nhà hàng (nơi giao)
    private String customerAddress; // Địa chỉ khách hàng (nơi nhận)
}

File: frontend/36Food/app/src/main/java/com/example/a36food/data/dto/OrderResponseDTO.kt
package com.example.a36food.data.dto

import com.google.gson.annotations.SerializedName
import java.util.Date

data class OrderResponseDTO(
    val orderId: Long,
    val status: String?, // Đã xóa SerializedName và để status có thể null
    val orderDate: Date,
    val updatedAt: Date?,
    val totalAmount: Double,
    val itemsTotal: Double,
    val deliveryFee: Double,
    val note: String?,
    val restaurantId: String?,
    val restaurantAddress: String?,
    val customerAddress: String?,
    val shippingAddress: String?,
    val items: List<OrderItemDTO> = emptyList()
)

